src/FixerDefinition/FixerDefinitionInterface.php

```diff
@@ -35,7 +35,7 @@ public function getDescription();
     public function getConfigurationDescription();
 
     /**
-     * @return null|array $defaultConfiguration     null for non-configurable fixer
```

Comment:
agreed :+1: ,
I made comment for myself to check on later for another case; the multiple spaces between:
```$defaultConfiguration``` and ```null```
which I think is better to be a single space as there is nothing else in the PHPDoc to align with.